.circleci/config.yml

```diff
@@ -121,18 +121,9 @@ jobs:
       - run:
           name: Run UI tests against the latest stable IntelliJ version
           command: ./gradlew -Pheadless -Pagainst=latestStable uiTest
-      - when:
-          condition:
-            # Practice shows that the risk of UI tests failing on `latestMinorsOfOldSupportedMajors`
-            # once both binary compatibility checks and UI tests for latest versions passed is close to zero.
-            # Hence, to speed up the builds, we only run them for develop/master/hotfix/release and not on PRs.
-            matches:
-              pattern: "^(develop|hotfix/.+|master|release/.+)$"
-              value: << pipeline.git.branch >>
-          steps:
-            - run:
-                name: Run UI tests against earlier supported major IntelliJ versions
-                command: ./gradlew -Pheadless -Pagainst=latestMinorsOfOldSupportedMajors uiTest
+      - run:
```

Comment:
I'd even retain this entire `when`/`condition` machinery, just changing the `pattern` to:
```yaml
 pattern: "^(develop|hotfix/.+|master|release/.+|.*ui-test.*)$"
```

Reply:
who will remember that? 🙄

Reply:
I will ;)